src/app/components/pages/PostsIndex.jsx

```diff
@@ -106,10 +105,18 @@ class PostsIndex extends React.Component {
             account_name, // TODO: for feed
             order,
             posts,
+            username,
         } = this.props;
 
+        const status = this.props.status
+            ? this.props.status.getIn([category || '', order])
+            : null;
+        let fetching = (status && status.fetching) || this.props.loading;
+
         let emptyText = '';
-        if (order === 'feed') {
+        if (!process.env.BROWSER) {
```

Comment:
Hmm.. I'm not sure about this, wouldn't it prevent SSR rendering of any post lists?

Reply:
Ah good point, yes, I suggest:
```
if (!process.env.BROWSER && category === 'my') {
  fetching = true;
}
```